src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -95,5 +106,11 @@ void OnFocusedChange(object? sender, FocusChangeEventArgs e)
 			if (!e.HasFocus)
 				VirtualView?.Completed();
 		}
+
+		void OnKeyboardBackPressed(object? sender, EventArgs eventArgs)
+		{
+			VirtualView?.Completed();
+			PlatformView?.ClearFocus();
```

Comment:
I don't think we want to influence the focus ourselves here. I'm guessing this code was fixing a probably incorrectly in XF

Reply:
Updated.